lib/dynamic.js

```diff
@@ -46,7 +46,11 @@ Dynamic.converters = [];
 
 Dynamic.define = function(name, converter) {
   converter.typeName = name;
-  this.converters.unshift(converter);
+  this.converters[name] = function(val, ctx) {
+    // Ignore null/undefined
+    if (name !== 'array' && val === null || val === undefined) return val;
```

Comment:
I should probably expand the comment here - I skip `'array'` so that the `'array'` converter has a chance to coerce `null/undefined` to `[]`.